dictionaries/en_shared/src/README.md

```diff
@@ -8,5 +8,7 @@ If the words you want to add is valid for:
 
 - all English flavors (en_US, en_AU, en_CA, en_GB, ...),
   add them to [shared-additional-words.txt](https://github.com/streetsidesoftware/cspell-dicts/blob/main/dictionaries/en_shared/src/shared-additional-words.txt)
+- all American English (en_US)
```

Comment:
-ize words are not American. American's chose to use them over -ise words, but the truth is that -ize words originated in British English. The Oxford English dictionary prefers -ize over -ise. Which is why the original British English dictionary included -ize variants. 

This file should only have words shared by en_US and en_GB but are not in the en_GB-ise dictionary. For example, it should not contain "marshaling", because that is not a valid British spelling.

Reply:
Thanks for the context.

So let me rephrase the situation.

- The idea of the `en_shared/src/shared-additional-words-ize.txt`  is valid.
- The name `shared-additional-words-ize.txt` is fine for this list.
- the comment in the readme and the header of this file should be reworded. I rephrase what you wrote in your comment.

Here are the conclusions I come to with my understanding:
- the en_GB, en_AU, en_US, en_CA and en_GB-MIT dictionaries SHOULD include `en_shared/src/shared-additional-words-ize.txt` 
- the en_GB-ise dictionnaries MUST exclude the words from `en_shared/src/shared-additional-words-ize.txt` 

Please confirm, then I will perform the needed changes.

Reply:
Close. Thank you for clarifying.

> Here are the conclusions I come to with my understanding:
> 
> * the en_GB, en_AU, en_US, en_CA and en_GB-MIT dictionaries SHOULD include `en_shared/src/shared-additional-words-ize.txt`

- Only en_US and en_GB should use `shared-additional-words-ize.txt`. I don't want to add `-ize` words to the other dictionaries.

> * the en_GB-ise dictionnaries MUST exclude the words from `en_shared/src/shared-additional-words-ize.txt`

- It should just not add them, not exclude them. There are valid `-ize` words in the en_GB-ise dictionary. I would not want to automatically exclude them.

Reply:
Additional questions about `en_shared/src/shared-additional-words-ise.txt` file now.

I can see the words are added to all English dictionaries except the en_US one: so en_GB, en_AU, en_CA, en_GB-ise, en_GB-MIT.

But the words in the `en_shared/src/shared-additional-words-ise.txt` file might be directly in the en_US dictionary.

So here are my questions:
- what is the purpose of  `en_shared/src/shared-additional-words-ise.txt` file ?
- why isn't `en_shared/src/shared-additional-words-ise.txt` included in `en_US` ?
- shouldn't we exclude words in `en_shared/src/shared-additional-words-ise.txt` from en_US ?

Based on this file purpose, we might find better namings for -ize and -ise files.

Reply:
> - Only en_US and en_GB should use `shared-additional-words-ize.txt`. I don't want to add `-ize` words to the other dictionaries.
> 
> > * the en_GB-ise dictionnaries MUST exclude the words from `en_shared/src/shared-additional-words-ize.txt`
> 
> - It should just not add them, not exclude them. There are valid `-ize` words in the en_GB-ise dictionary. I would not want to automatically exclude them.

OK, it's way clearer now. I can work from that.

Reply:
This thread also have interesting things to gather in a README file

Reply:
I opened https://github.com/streetsidesoftware/cspell-dicts/pull/4487

- https://github.com/streetsidesoftware/cspell-dicts/pull/4487